module/zfs/dsl_scan.c

```diff
@@ -3173,12 +3213,16 @@ dsl_scan_sync(dsl_pool_t *dp, dmu_tx_t *tx)
 	spa_t *spa = dp->dp_spa;
 	state_sync_type_t sync_type = SYNC_OPTIONAL;
 
+	if (spa->spa_resilver_defered &&
+	    !spa_feature_is_active(dp->dp_spa, SPA_FEATURE_RESILVER_DEFER))
+		spa_feature_incr(spa, SPA_FEATURE_RESILVER_DEFER, tx);
```

Comment:
What is the refcount of the feature?  It seems like it's incremented at most once, but decremented for every deferred vdev.

Reply:
It is not decremente for every vdev. `dsl_scan_reset_defered()` only resets it once it processed the rootvd

Reply:
oh, I see.